control_panel_api/aws.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+import boto3
+
+
+def create_bucket(name, region, **kwargs):
+    boto3.client('s3').create_bucket(
+        Bucket=name,
+        ACL=kwargs.get('acl', 'private'),
+        CreateBucketConfiguration={'LocationConstraint': region},
+    )
+
+
+def delete_bucket(name):
+    boto3.client('s3').delete_bucket(Bucket=name)
+
+
+def put_bucket_logging(name, target_bucket, target_prefix):
+    boto3.client('s3').put_bucket_logging(
+        Bucket=name,
+        BucketLoggingStatus={
+            'LoggingEnabled': {
+                'TargetBucket': target_bucket,
+                'TargetPrefix': target_prefix
+            }
+        }
+    )
+
+
+def create_policy(name, policy_document):
+    boto3.client("iam").create_policy(
+        PolicyName=name,
+        PolicyDocument=json.dumps(policy_document)
+    )
+
+
+def delete_policy(policy_arn):
+    boto3.client("iam").delete_policy(PolicyArn=policy_arn)
+
+
+def detach_policy_from_entities(policy_arn):
```

Comment:
As quickly discussed offline, I think until we really need to perform this operation individually we may just make this function "private" (rename to `_detach....`) and call always in `delete_policy()` before the actual deletion of the policy.

(I would imagine from the service level perspective this is an implementation detail, the client code doesn't really concern with whether you need to detach the policy from stuff or not in order to delete it)